SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorModule.cpp

```diff
@@ -21,9 +21,13 @@
 #include "SpatialGDKEditorSettings.h"
 #include "SpatialGDKSettings.h"
 #include "SpatialLaunchConfigCustomization.h"
+#include "SpatialGDKFunctionalTests/Public/SpatialFunctionalTest.h"
```

Comment:
I think we try to sort our includes alphabetically (though we are not always consistent)
Or separating engine and GDK includes.

Reply:
I did it afterwards to be together with all Spatial ones, but this one is from the other module. If you prefer it moved I can

Reply:
Yeah just throw a line break between gdk and engine includes